crates/vm/levm/src/opcode_handlers/stack_memory_storage_flow.rs

```diff
@@ -262,7 +282,11 @@ impl VM {
             .try_into()
             .unwrap_or(usize::MAX);
 
-        let words_copied = (size + WORD_SIZE - 1) / WORD_SIZE;
+        let words_copied = (size
+            .checked_add(WORD_SIZE)
+            .ok_or(VMError::VeryLargeNumber)?
+            .saturating_sub(1))
+            / WORD_SIZE;
```

Comment:
Shouldn't we use `checked_div` here?

Reply:
`checked_div` only checks if denominator is zero, and WORD_SIZE is a non-zero constant